sphinx/domains/std.py

```diff
@@ -336,6 +335,7 @@ class Glossary(SphinxDirective):
     def run(self) -> List[Node]:
         node = addnodes.glossary()
         node.document = self.state.document
+        node.sorted = ('sorted' in self.options)
```

Comment:
We usually use node-attributes to store data. How about using it instead?
```suggestion
        node["sorted"] = ('sorted' in self.options)
```